Assignment4/src/test/java/com/example/Assignment4/ItemTest.java

```diff
@@ -0,0 +1,89 @@
+package com.example.Assignment4;
+
+
+import com.example.Assignment4.entity.Items;
+import com.example.Assignment4.entity.ItemsBuilder;
+import com.example.Assignment4.repository.ItemRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.annotation.Rollback;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+
+/*
+ * It is use to test CRUD operation in database
+ */
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public class ItemTest {
+
+    /*
+     *injection of item repository
+     */
+    @Autowired
```

Comment:
why are you autowiring ? We discussed that u should use mockito library to write Junit test cases .